src/solver/conv_asm_implicit_gemm_gtc_bwd_nhwc.cpp

```diff
@@ -777,6 +777,11 @@ bool PerformanceConfigAsmImplicitGemmGTCBwdXdlopsNHWC::IsValid(const Convolution
         if(ctx.IsFp16() && gemm_k_global_split != 0 && vector_store != 1)
             return false;
 
+    // we always have 2 configs that are the same expect gemm_k_global_split=1 and 0
+    // so it's safe to disable the one with gemm_k_global_split=1 here
+    if(miopen::IsEnabled(MIOPEN_DEBUG_CONVOLUTION_DETERMINISTIC{}) && gemm_k_global_split != 0)
```

Comment:
[Recommendation] Please fully explain the machinery in comments. You can re-use text from https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1377#discussion_r790292417 and from https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1377#discussion_r790705199. Otherwise I'll need to open a post-merge issue.

Reply:
@atamazov OK I update the comments here.